src/test/store/__snapshots__/transforms.test.js.snap

```diff
@@ -70,3 +70,97 @@ exports[`focus subtree transform on an inverted call tree starts as an inverted
      - B (total: 1.0ms, self:0.0ms)
        - A (total: 1.0ms, self:0.0ms)"
 `;
+
```

Comment:
It's probably a pain, but I would recommend looking at the tests along with these fixtures to verify that my assumptions were correct. I'm not happy that the assertions are separated, but the CLI diffs are too dang nice when things aren't working correctly. I tried inlining the text into the tests, but it was really fiddly, and the errors weren't nearly as nice.